plenario/api/point.py

```diff
@@ -88,19 +46,20 @@ def detail_aggregate():
     validator = NoGeoJSONDatasetRequiredValidator(only=fields)
     validated_args = validate(validator, request.args.to_dict())
     if validated_args.errors:
-        return bad_request(validated_args)
+        return bad_request(validated_args.errors)
     return _detail_aggregate(validated_args)
 
 
 @cache.cached(timeout=CACHE_TIMEOUT, key_prefix=make_cache_key)
 @crossdomain(origin="*")
 def detail():
     fields = ('location_geom__within', 'dataset_name', 'shape', 'obs_date__ge',
-              'obs_date__le', 'data_type', 'offset')
-    validator = NoGeoJSONDatasetRequiredValidator(only=fields)
+              'obs_date__le', 'data_type', 'offset', 'date__time_of_day_ge',
```

Comment:
/detail was not considering hour params because I had left date__time_of_day out of this tuple